services/middlewareServices/AuthMiddlewareService.js

```diff
@@ -0,0 +1,102 @@
+// Import logger
+const logger = require("@logger/logger")
+
+// Import errors
+const { AuthError } = require("@errors/AuthError")
+
+const jwtUtils = require("@utils/jwt-utils")
+
+const { BadRequestError } = require("@root/errors/BadRequestError")
+
+const { E2E_TEST_REPO, E2E_TEST_SECRET, E2E_TEST_GH_TOKEN } = process.env
+const E2E_TEST_USER = "e2e-test"
+const GENERAL_ACCESS_PATHS = ["/v1/sites", "/v1/auth/whoami"]
+
+class AuthMiddlewareService {
+  verifyE2E({ cookies, url }) {
+    const { isomercmsE2E } = cookies
+    const urlTokens = url.split("/") // urls take the form "/v1/sites/<repo>/<path>""
+    let isValidE2E
```

Comment:
```js
const isValidE2E = false
```

Reply:
Refactored to return early in https://github.com/isomerpages/isomercms-backend/pull/328/commits/f67202b7ee8461090aff2746ddcf0652b5c69251